src/Validate.hs

```diff
@@ -194,6 +194,38 @@ checkDefinition (Valid.Definition pattern body _) =
                 Result.throw (R.merge start end) (Error.BadFunctionName (length args))
 
 
+
+checkArguments :: [Pattern.Pattern R.Region var] -> Result.Result wrn Error.Error ()
```

Comment:
I guess there a type synonym for "Pattern.Pattern R.Region var" but I couldn't find it.